tests/e2e/pageObjects/base-run-commands-page.ts

```diff
@@ -0,0 +1,78 @@
+    copyBtn = Selector('[data-testid^=copy-btn-]');
+    copyCommand = Selector('[data-testid=copy-command]');
+
+    runButtonToolTip = Selector('[data-testid=run-query-tooltip]');
+    loadedCommand = Selector('[class=euiLoadingContent__singleLine]');
+    runButtonSpinner = Selector('[data-testid=loading-spinner]');
+    commandExecutionDateAndTime = Selector('[data-testid=command-execution-date-time]');
+    executionCommandTime = Selector('[data-testid=command-execution-time-value]');
+    executionCommandIcon = Selector('[data-testid=command-execution-time-icon]');
+    executedCommandTitle = Selector('[data-testid=query-card-tooltip-anchor]', { timeout: 500 });
+    queryResult = Selector('[data-testid=query-common-result]');
+
+    cssQueryCardCommand = '[data-testid=query-card-command]';
+    cssQueryTextResult = '[data-testid=query-cli-result]';
+    cssReRunCommandButton = '[data-testid=re-run-command]';
+    cssDeleteCommandButton = '[data-testid=delete-command]';
+
+    getTutorialLinkLocator = (tutorialName: string): Selector =>
+        Selector(`[@data-testid="data-testid=query-tutorials-link_${tutorialName}"]`);
+
+    /**
+     * Get card container by command
+     * @param command The command
+     */
+    async getCardContainerByCommand(command: string): Promise<Selector> {
+        return this.queryCardCommand.withExactText(command).parent('[data-testid^="query-card-container-"]');
+    }
+
+    /**
+     * Send a command in Workbench
+     * @param command The command
+     * @param speed The speed in seconds. Default is 1
+     * @param paste
+     */
+    async sendCommandInWorkbench(command: string, speed = 1, paste = true): Promise<void> {
+        await t
+            .click(this.queryInput)
+            .typeText(this.queryInput, command, { replace: true, speed, paste })
+            .click(this.submitCommandButton);
+    }
+
+    /**
+     * Check the last command and result in workbench
+     * @param command The command to check
+     * @param result The result to check
+     * @param childNum Indicator which command result need to check
+     */
+    async checkWorkbenchCommandResult(command: string, result: string, childNum = 0): Promise<void> {
+        // Compare the command with executed command
+        const actualCommand = await this.queryCardContainer.nth(childNum).find(this.cssQueryCardCommand).textContent;
+        await t.expect(actualCommand).contains(command, 'Actual command is not equal to executed');
+        // Compare the command result with executed command
+        const actualCommandResult = await this.queryCardContainer.nth(childNum).find(this.cssQueryTextResult).textContent;
+        await t.expect(actualCommandResult).contains(result, 'Actual command result is not equal to executed');
+    }
+
+    /**
+     * Get selector with tutorial name
+     * @param tutorialName name of the uploaded tutorial
+     */
+    getAccordionButtonWithName(tutorialName: string): Selector {
```

Comment:
lets store this method as Selector like getTutorialLinkLocator above

Reply:
remove the selector since it is not used